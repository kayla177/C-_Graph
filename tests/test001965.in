ENTITY UQSBP0FQ name_UQSB conference
ENTITY LZ8L5QJ4 name_LZ8L institution
ENTITY 5YY4VB2D name_5YY4 institution
ENTITY JTUCH3KI name_JTUC author
ENTITY 82RD7MIA name_82RD journal
ENTITY B5B17PJE name_B5B1 conference
ENTITY NCGAHPWD name_NCGA author
ENTITY DS7D3PWN name_DS7D author
ENTITY 6CU9XBI2 name_6CU9 paper
ENTITY 1DCFWR9S name_1DCF author
RELATIONSHIP 6CU9XBI2 authored_by 1DCFWR9S 9.4
RELATIONSHIP 82RD7MIA authored_by 6CU9XBI2 0.7
RELATIONSHIP 1DCFWR9S affiliated_with JTUCH3KI 4.8
RELATIONSHIP DS7D3PWN collaborated_with 1DCFWR9S 8.4
RELATIONSHIP UQSBP0FQ published_in B5B17PJE 1.1
RELATIONSHIP UQSBP0FQ cited_by 5YY4VB2D 6.7
RELATIONSHIP DS7D3PWN authored_by UQSBP0FQ 4.2
RELATIONSHIP UQSBP0FQ cited_by NCGAHPWD 0.9
RELATIONSHIP 5YY4VB2D cited_by NCGAHPWD 6.7
RELATIONSHIP DS7D3PWN authored_by 82RD7MIA 2.6
RELATIONSHIP NCGAHPWD presented_at JTUCH3KI 3.5
RELATIONSHIP UQSBP0FQ authored_by JTUCH3KI 2.5
RELATIONSHIP 82RD7MIA affiliated_with B5B17PJE 1.5
RELATIONSHIP 5YY4VB2D authored_by B5B17PJE 0.2
RELATIONSHIP DS7D3PWN presented_at 5YY4VB2D 4.5
PRINT B5B17PJE
PRINT JTUCH3KI
HIGHEST
DELETE B5B17PJE
HIGHEST
HIGHEST
PRINT 5YY4VB2D
PATH DS7D3PWN 5YY4VB2D
PATH 6CU9XBI2 JTUCH3KI
PATH 1DCFWR9S JTUCH3KI
EXIT
