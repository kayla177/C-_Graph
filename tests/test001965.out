success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
UQSBP0FQ 82RD7MIA 5YY4VB2D
1DCFWR9S NCGAHPWD UQSBP0FQ
NCGAHPWD 6CU9XBI2 29
success
NCGAHPWD 6CU9XBI2 29
NCGAHPWD 6CU9XBI2 29
UQSBP0FQ NCGAHPWD DS7D3PWN
DS7D3PWN 5YY4VB2D 4.5
6CU9XBI2 1DCFWR9S JTUCH3KI 14.2
1DCFWR9S JTUCH3KI 4.8
